src/components/Footer.jsx

```
@@ -0,0 +1,141 @@
+    { title: "Sign Up", path: "/signup" },
+    { title: "Login", path: "/login" },
+    { title: "Forgot Password", path: "/forgot-password" }
+  ];
+
+  const universityLinks = [
+    { title: "University Sign Up", path: "/university/signup" },
+    { title: "University Login", path: "/university/login" },
+    { title: "University Dashboard", path: "/university/dashboard" }
+  ];
+
+  const adminLinks = [
```

ESLint

[error] 'adminLinks' is assigned a value but never used.

(no-unused-vars)